plugins/pci-mei/fu-plugin-pci-mei.c

```diff
@@ -99,6 +193,32 @@ fu_plugin_add_security_attrs_override_strap (FuPlugin *plugin, FuSecurityAttrs *
 	fwupd_security_attr_set_result (attr, FWUPD_SECURITY_ATTR_RESULT_LOCKED);
 }
 
+static void
+fu_plugin_add_security_attrs_csme_version (FuPlugin *plugin, FuSecurityAttrs *attrs)
+{
+	FuPluginData *priv = fu_plugin_get_data (plugin);
+	g_autoptr(FwupdSecurityAttr) attr = NULL;
+
+	if (priv->issue == FU_MEI_ISSUE_UNKNOWN)
+		return;
```

Comment:
I think a warning into syslog makes sense here on the logic that an unknown ME architecture needs to start getting tracked too and this will cause some noise to be stirred.

Reply:
syslog, or just fail the HSI test?

Reply:
Hmm, I think I'm swayed more towards fail the syslog approach.  A "new" ME architecture is very likely to include all the previous hotfixes.  By the time a vulnerability came out it should have more details.

Reply:
I've added a g_warning which should make it into the logs. I also saved the FuMeiVersion so the log message would be half-useful.